### PR TITLE
Improve message for no docstring

### LINKED ISSUE #2234: Object Inspector is not showing "No documentation available" for objects without docstring
It seems Jedi returns `<no docstring>` instead of an empty string for objects without docstring. This prevents the Object Inspector to show the *No documentation available* warning page.

### PR DESCRIPTION
**Pull request by blink1073**
Fixes #2234.

### CODE REVIEW

**Comment by Nodd** [comment]
You're fast !
The sentence should be translatable (is it a word?), though.

**Comment by blink1073** [comment]
I barely speak my own language, I leave that to you polyglots.   :smile:

**Comment by Nodd** [comment]
No, I mean it should use spyder's system to declare that sentence as translatable, so that polyglots can include the translation.
I don't remember the details now, but you sumply have to put the string between ```_(``` and  ```)```, and add an import if it's not already there.

**Comment by blink1073** [comment]
Oh, ha!

**Comment by blink1073** [comment]
All set!

**Comment by ccordoba12** [comment]
I don't like it. Could you just leave what we had before?

**Comment by ccordoba12** [comment]
The thing is the *No documentation available* page will not be shown again because of this change

**Comment by blink1073** [comment]
What don't you like about it?  Let's say the function was in another file.  We'd know it was a function, and what module it was in.  The other way, we might think it does not even exist.

**Comment by blink1073** [comment]
I can make it so that page is shown where there is indeed nothing found.

**Comment by ccordoba12** [comment]
I'd prefer that the *No doc available* message (in red and everything) would be shown below the signature.

**Comment by blink1073** [comment]
Better?

screenshot from 2015-03-13 16 25 52

**Comment by Nodd** [comment]
:+1: Shiny !

File: spyderlib/utils/introspection/plugin_manager.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 from spyderlib.qt.QtCore import Signal, QThread, QObject, QTimer
 
 
-PLUGINS = ['rope', 'jedi', 'fallback']
+PLUGINS = ['jedi', 'rope', 'fallback']
```
Comment (ccordoba12) [inline spyderlib/utils/introspection/plugin_manager.py]: Please revert this change.

**Comment by ccordoba12** [comment]
**Much** better. I just have another suggestion: could you add an horizontal line between the signature and the message?

**Comment by blink1073** [comment]
Now:

screenshot from 2015-03-13 21 31 54

**Comment by ccordoba12** [comment]
Yes, thanks a lot! I'll merge it tomorrow :-)

File: spyderlib/plugins/inspector.py [hunk 2]
```diff
@@ -21,7 +21,7 @@
 
 # Local imports
 from spyderlib import dependencies
-from spyderlib.baseconfig import get_conf_path, get_module_source_path, _
+from spyderlib.baseconfig import get_conf_path, get_module_source_path, _, debug_print
```
Comment (ccordoba12) [inline spyderlib/plugins/inspector.py]: `debug_print` is not used. Is it a left over?

File: spyderlib/utils/introspection/rope_plugin.py [hunk 3]
```diff
@@ -167,7 +167,7 @@ def handle_info(self, cts, doc_text, source_code, offset):
                     calltip = getsignaturefromtext(doc_text, obj_name)
         if not obj_fullname:
             obj_fullname = sourcecode.get_primary_at(source_code, offset)
-        if obj_fullname and not obj_fullname.startswith('self.') and doc_text:
```
Comment (ccordoba12) [inline spyderlib/utils/introspection/rope_plugin.py]: Why did you remove this condition?

File: spyderlib/plugins/inspector.py [hunk 4]
```diff
@@ -334,6 +333,10 @@ def run(self):
                                                math=self.math_option,
                                                img_path=self.img_path)
                     html_text = sphinxify(doc['docstring'], context)
+                    if doc['docstring'] == '':
+                        html_text += '<hr>'
```
Comment (ccordoba12) [inline spyderlib/plugins/inspector.py]: Please change this line to

    html_text += '<div class="hr"></div>'

to better style this hr

File: spyderlib/utils/introspection/rope_plugin.py (hunk 3 above)
Comment (blink1073) [inline spyderlib/utils/introspection/rope_plugin.py]: This allows us to provide the other information while still showing the "No documentation" banner.

**Comment by blink1073** [comment]
@ccordoba12, fixed.
